Blizzard.IO.RabbitMQ.sln

```diff
@@ -7,6 +7,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Blizzard.IO.RabbitMQ", "Bli
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Blizzard.IO.Core", "Blizzard.IO.Core\Blizzard.IO.Core\Blizzard.IO.Core.csproj", "{D2111278-8B66-43F4-8C69-71BDD9AB540E}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "SubModules", "SubModules", "{7BD0B07F-CFEA-4AB7-9D42-A84278157B7A}"
```

Comment:
I think we have to wait for our nuget server in order to work properly
We donwt want these `SubModules` to merge into the project

Reply:
We dont know when we will have the server,
we can delete this directory at any time and convert this to a nuget package reference.